app/models/views/user/statement_overview.rb

```diff
@@ -6,49 +6,75 @@ class StatementOverview
       # include PublishersHelper
 
       GROUP_START_DATE = Date.new(2019, 10, 1)
+      MONTH_FORMAT = "%b %Y".freeze
+      YEAR_FORMAT = "%b %e, %Y".freeze
```

Comment:
I think we should use a standard date format since we've never discussed this. I'm a fan of ISO 8601 which is YYYY-MM-DD or `%Y-%m-%d`

Reply:
This class has been kind of treated as a decorator for the view, but I'd be happy to not do any formatting and put the formatting in the React code if that would make more sense?

Reply:
https://github.com/brave-intl/publishers/pull/2638/commits/4ca571bca6e8fc5ec8c8a9bd9694bde98804bce1